### PR TITLE
HPCC-14900 ConfigMgr - Fix do_not_generate_list in genenvrules.conf

### PR DESCRIPTION
**Pull request by garonsky**
Signed-off-by: Gleb Aronsky gleb.aronsky@lexisnexis.com

### CODE REVIEW

**Comment by garonsky** [comment]
@wangkx please review

**Comment by garonsky** [comment]
@xwang2713

**Comment by wangkx** [comment]
@garonsky looks fine.

File: initfiles/etc/DIR_NAME/genenvrules.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 [Algorithm]
 max_comps_per_node=4
-do_not_generate=SiteCertificate,dfuplus,soapplus,eclplus,ldapServer,ws_account,eclserver,ecldirect,DynamicESDL,ESPLoggingAgent,LoggingManager,WsLogging
+do_not_generate=SiteCertificate,dfuplus,soapplus,eclplus,ldapServer,ws_account,eclserver,ecldirect,DynamicESDL,esploggingagent,loggingmanager,wslogging
```
Comment (richardkchapman) [inline initfiles/etc/DIR_NAME/genenvrules.conf]: Dies DynamocESDL also need to be all lower-case?

Comment (garonsky) [inline initfiles/etc/DIR_NAME/genenvrules.conf]: No.  DynamicESDL is capitalized in the buildset:  https://github.com/hpcc-systems/HPCC-Platform/blob/candidate-6.0.0/initfiles/componentfiles/configxml/buildsetCC.xml.in#L323

**Pull request merged**